### PR TITLE
Fixes VSTS Bug 638359: Update JS/TS tmLanguage files

### PR DESCRIPTION
**Pull request by mkrueger**
Fixes issue #5212 Show scopes command includes markup.
Fixes VSTS Bug 595108: [Feedback] VS for MAC 7.4 and 7.5 preview: javascript syntax highlighting breaks when passing parameter to a function

### CODE REVIEW

**Comment by mkrueger** [comment]
@monojenkins backport release-7.6

**Comment by slluis** [comment]
@KirillOsenkov can you please verify that the bug is fixed? thanks.

**Comment by slluis** [comment]
@mkrueger the test RegexTests.TestUsingMatch failed.

**Comment by mkrueger** [comment]
Fixed the test - ironically this test tested that the bug works.

**Comment by KirillOsenkov** [comment]
cc @minestarks

**Comment by KirillOsenkov** [comment]
I can confirm this fixes the bug - thanks Mike!

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Editor.Highlighting/VSCodeImport/typescript/syntaxes/TypeScriptReact.tmLanguage.json [hunk 1]
```diff
@@ -4,13 +4,9 @@
...
 		"Once accepted there, we are happy to receive an update request."
 	],
-	"version": "https://github.com/Microsoft/TypeScript-TmLanguage/commit/3a70fab1b03520774fa236f1f7e7a0939463739f",
+	"version": "https://github.com/Microsoft/TypeScript-TmLanguage/commit/7bf8960f7042474b10b519f39339fc527907ce16",
 	"name": "TypeScriptReact",
 	"scopeName": "source.tsx",
-	"fileTypes": [
-		"tsx"
-	],
-	"uuid": "805375ec-d614-41f5-8993-5843fe63ea82",
```
Comment (KirillOsenkov) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Editor.Highlighting/VSCodeImport/typescript/syntaxes/TypeScriptReact.tmLanguage.json]: @mkrueger this removes the fileTypes for .tsx - will it still work?

**Pull request merged**